yesod-test/Yesod/Test.hs

```diff
@@ -329,6 +331,13 @@ getTestYesod = fmap yedSite getSIO
 getResponse :: YesodExample site (Maybe SResponse)
 getResponse = fmap yedResponse getSIO
 
+-- | Get the cookies set for the current test. Differs from `getRequestCookies`
+-- in that this allows you to access cookies outside of a request builder
+--
+-- Since 1.6.16
```

Comment:
```suggestion
-- @since 1.6.16
```

Reply:
Oh, looks like 1.6.16 was already released, so needs to be 1.6.17

Reply:
Bumped!